piccolo_api/jwt_auth/middleware.py

```diff
@@ -96,9 +82,9 @@ async def __call__(self, scope, receive, send):
         if await self.blacklist.in_blacklist(token):
             raise HTTPException(status_code=403, detail="Token revoked")
 
-        token_dict = jwt.decode(token, self.secret)
-
-        if self.has_expired(token_dict):
+        try:
+            token_dict = jwt.decode(token, self.secret, algorithms=["HS256"])
```

Comment:
I think it's safe to remove `expiry` from the JWT payload, since that's already done by JWT and will be handled by that,

Reply:
You're right, good catch. I just checked the docs:

https://pyjwt.readthedocs.io/en/latest/usage.html#expiration-time-claim-exp